flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -172,20 +172,22 @@ object AggregateUtil {
     * @param namedAggregates    List of calls to aggregate functions and their output field names
     * @param inputRowType       Input row type
     * @param inputFieldTypes    Types of the physical input fields
+    * @param outputType         Output type of the aggregate node
```

Comment:
`Output type of the aggregate node` -> `Output type of the (table)aggregate node` ？ please check all the JavaDoc，for aggregate and tableAggregate， and make the consistency of the descript.